eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         displayName: "Component Detection"
         # ComponentGovernance is currently unable to run on pull requests of public projects.  Running on
         # scheduled builds should be good enough.
+        ignoreDirectories: "sdk\storage\Azure.Storage.Common\swagger\Generator"
```

Comment:
should we use `/` for the path separators?

Reply:
I think it can handle both but we should make ```/``` the convention unless something breaks. I'll update the PR.